RoadMap_VK/RoadMap_VK/Controllers/PhotoViewController.swift

```diff
@@ -0,0 +1,45 @@
+// PhotoViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Экран показа фото
+final class PhotoViewController: UIViewController {
+    // MARK: - Public properties
+
+    var photoNames: [String] = []
+
+    // MARK: - Private properties
```

Comment:
```suggestion
    // MARK: - Private Visual Components
```
мне кажется эта марка сюда больше подходит